NEWS.md

```diff
@@ -1,5 +1,8 @@
 # dbplyr (development version)
 
+* When possible, dbplyr now uses `SELECT *` after a left join instead of
```

Comment:
Or inner join?

Could we apply the same logic to right joins too?

Reply:
Not that simple to extend this to all type of joins but now they are all supported (note that for `full_join()` one can never use `*`).